subxt/src/backend/unstable/follow_stream_driver.rs

```diff
@@ -379,6 +380,77 @@ struct SubscriberDetails<Hash: BlockHash> {
     waker: Option<Waker>,
 }
 
+pin_project! {
```

Comment:
Do we need this? I don't think anything needs to be pinned really, so I suspect you can just write something like:

```rust
impl Unpin for FollowStreamFinalizedHeads {}
```

Then, you can access self and do anything with 100% safe rust in the stream impl :)